src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -766,26 +763,26 @@ EscrowFinish::preclaim(PreclaimContext const& ctx)
             return err;
     }
 
-    auto const k = keylet::escrow(ctx.tx[sfOwner], ctx.tx[sfOfferSequence]);
-    auto const slep = ctx.view.read(k);
-    if (!slep)
-        return tecNO_TARGET;
```

Comment:
This error needs to be thrown in `doApply` in both transactions

Reply:
It is already thrown in doApply for Cancel & Finish. It wasn't "moved" but rather copied.